tox.ini

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't see this being used on the travis configuration, but maybe I'm missing something?